CHANGELOG.md

```diff
@@ -18,12 +18,70 @@ The release notes for the ROCm platform.
 ## ROCm 5.6.0
 <!-- markdownlint-disable first-line-h1 -->
 <!-- markdownlint-disable no-duplicate-header -->
+### What's New in This Release
+
+#### Change Log for HIP
+
+Full documentation for HIP is available at rocm.docs.amd.com
+
+##### HIP 5.6 (For ROCm 5.6)
+
+###### Optimizations
+
+- Consolidation of hipamd, rocclr and OpenCL projects in clr
+- Optimized lock for graph global capture mode
+
+###### Added
+
+- Added hipRTC support for amd_hip_fp16
+- Added hipStreamGetDevice implementation to get the device associated with the stream
+- Added HIP_AD_FORMAT_SIGNED_INT16 in hipArray formats
+- hipArrayGetInfo for getting information about the specified array
+- hipArrayGetDescriptor for getting 1D or 2D array descriptor
+- hipArray3DGetDescriptor to get 3D array descriptor
+
+###### Changed
+
+- hipMallocAsync to return success for zero size allocation to match hipMalloc
+- Separation of hipcc perl binaries from HIP project to hipcc project. hip-devel package depends on newly added hipcc package
+- Consolidation of hipamd, ROCclr, and OpenCL repositories into a single repository called clr. Instructions are updated to build HIP from sources in the HIP Installation guide
+- Removed hipBusBandwidth and hipCommander samples from hip-tests
+
+###### Fixed
+
+- Fixed regression in hipMemCpyParam3D when offset is applied
+
+###### Known Issues
+
+- Limited testing on xnack+ configuration
+  - Multiple HIP tests failures (gpuvm fault or hangs)
+
+###### Upcoming changes in future release
+
+- Removal of gcnarch from hipDeviceProp_t structure
+- Addition of new fields in hipDeviceProp_t structure
+  - maxTexture1D
+  - maxTexture2D
+  - maxTexture1DLayered
+  - maxTexture2DLayered
+  - sharedMemPerMultiprocessor
+  - deviceOverlap
+  - asyncEngineCount
+  - surfaceAlignment
+  - unifiedAddressing
+  - computePreemptionSupported
+  - uuid
+- Removal of deprecated code
+  - hip-hcc codes from hip code tree
+- Correct hipArray usage in HIP APIs such as hipMemcpyAtoH and hipMemcpyHtoA
+- HIPMEMCPY_3D fields correction (unsigned int -> size_t)
+- Renaming of 'memoryType' in hipPointerAttribute_t structure to 'type'
 
 ### Library Changes in ROCM 5.6.0
 
 | Library | Version |
 |---------|---------|
-| hipBLAS | [0.53.0](https://github.com/ROCmSoftwarePlatform/hipBLAS/releases/tag/rocm-5.6.0) |
+| hipBLAS | 0.54.0 ⇒ [1.0.0](https://github.com/ROCmSoftwarePlatform/hipBLAS/releases/tag/rocm-5.6.0) |
```

Comment:
I don't think that 5.6 updates hipBLAS version if you check with the [`release/rocm-rel-5.6`-branch](https://github.com/ROCmSoftwarePlatform/hipCUB/blob/release/rocm-rel-5.6/CHANGELOG.md).

Reply:
I think the auto tag script may have gotten it from this: https://github.com/ROCmSoftwarePlatform/hipBLAS/blob/release/rocm-rel-5.6/CHANGELOG.md

Reply:
Ah you're right. I'm not 100% sure if 1.0.0 is for 5.6.0, it might be for 5.7/6.0 instead. @saadrahim can you confirm?